### PR TITLE
Created A New Dedicated Contact Page

### PR DESCRIPTION
**Pull request by swarnade**
Created A New Dedicated Contact Page

### CODE REVIEW

**Comment by swarnade** [comment]
Closes Issue #718

**Comment by akshathar-navada** [comment]
Can u pls add required screenshots to review @swarnade

**Comment by swarnade** [comment]
image
image
image
This Are the Sample Image Have a Look

**Comment by akshathar-navada** [comment]
Hey, remove the 1st screenshot section. Becos the layout of that doen't look  good. Let only the contact section & footer & navbar be. And also change the title of the page, like it is showing OS-COde| chapters na...

File: contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,465 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+  <title>OS-CODE | Contact Us</title>
```
Comment (swarnade) [inline contact-us/contact-us.html:8]: Changed Title To Contact Us

**Review by swarnade — COMMENTED** [review]
Screenshot (1063)

**Comment by swarnade** [comment]
@akshathar-navada  Done All Changes According To Your Requirement.

**Comment by akshathar-navada** [comment]
Hey we need something like this, with proper layout , only navbar, contact section and the footer

image

image

**Pull request closed without merging**